ocs_ci/deployment/terraform.py

```diff
@@ -16,12 +16,16 @@ class Terraform(object):
     def __init__(self, path):
         self.path = path
 
-    def initialize(self):
+    def initialize(self, upgrade=None):
         """
         Initialize a working directory containing Terraform configuration files
```

Comment:
Please add it to docstring with explanation. Thanks

Reply:
Done